### ISSUE TITLE
Implement accessibility and navigation for Workspace

### ISSUE THREAD

**Issue by chinakov** [audacity/audacity#8958]
### Description

Currently, the 'Workspace' selector in the toolbar is not accessible via keyboard shortcuts. Let's make it and the subsequent buttons accessible via keyboard.

![Image](https://github.com/user-attachments/assets/fd398640-3a0d-4ebf-ad7c-df3ce7db8471)

It also requires navigation definition for the navigation tree. This is something for the developer to implement.

**Comment by Eism** [comment]
was fixed in https://github.com/audacity/audacity/pull/10116

**Comment by chinakov** [comment]
This has been fixed indeed. Thanks!
